dashboard/app/views/levels/_external_link.html.haml

```diff
@@ -10,7 +10,7 @@
   .external-link-disclaimer
     %h2= t('external_links.disclaimer')
     %strong= t('external_links.leaving_domain')
-    %p!= t('external_links.disclaimer_text', support_url: 'https://support.code.org')
+    %p!= t('external_links.disclaimer_text', support_url: 'https://support.code.org', markdown: true)
```

Comment:
nit: this should probably be either

```suggestion
    %p!= t('external_links.disclaimer_text', support_url: 'https://support.code.org', markdown: :inline)
```

or

```suggestion
    != t('external_links.disclaimer_text', support_url: 'https://support.code.org', markdown: true)
```

the difference between the two being that the regular case means the markdown will handle its own block-level elements like paragraphs and lists and whatnot, while the `inline` case will produce content suitable for embedding inside an existing block-level element.